src/components/Editor.js

```diff
@@ -90,6 +92,36 @@ const Editor = React.createClass({
     this.toggleBreakpoint(line);
   },
 
+  onContextMenu(cm, event) {
+    event.stopPropagation();
+    event.preventDefault();
+
+    const { line, ch } = this.editor.codeMirror.coordsChar({
+      left: event.clientX,
+      top: event.clientY
+    });
+
+    const sourceLocation = {
+      sourceId: this.props.selectedLocation.sourceId,
+      line: line++,
+      column: ch++
+    };
+
+    const pairedType = isOriginalId(this.props.selectedLocation.sourceId)
+      ? L10N.getStr("generated") : L10N.getStr("original");
+
+    const jumpLabel = {
+      accesskey: "C",
```

Comment:
I believe this needs to be either a `G` or an `O` and correspond to the label string used.  Though I guess we aren't localizing these...  

It might be simpler to create both labels and then choose which one to add to the menu below.